todo_app/templates/index.html

```diff
@@ -96,8 +96,8 @@ <h1 style="display:inline"></h1>
         {% for task in view_model.show_all_done_items %}
         <tr>
           <td style="display:none">{{ task.id }}</td>
-          <td>{{ task.title }}</td>
-          <td>{{ task.due }}</td>
+          <td>{{ task.name }}</td>
+          <td>{{ (task.due).strftime("%Y-%m-%d") }}</td>
```

Comment:
It might be nicer to build this formatting into the task itself, i.e. card could have a property called `date` that converts like this for you

Reply:
TY. Updated in Mod 12.